src/server.c

```diff
@@ -3563,6 +3578,7 @@ void call(client *c, int flags) {
         char *latency_event = (real_cmd->flags & CMD_FAST) ?
                                "fast-command" : "command";
         latencyAddSampleIfNeeded(latency_event,duration/1000);
+        durationAddSample("command", duration, 0);
```

Comment:
I believe this double counts multi-commands, since it will count the outer and inner time,